kedro-org/kedro-website
# Add Slack logo and link

Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Resolves #66.

## Development notes

I've added the Slack logo svg and used a workspace invite link (for now) as the link's address.

## QA notes

View the deploy preview and make sure the logo looks good and links to the right place.

## Still to consider

We need to figure out a way to get around the "400 person limitation" and "expires in 30 days" limits on links so that we don't have to update the link.

## Checklist
